### ISSUE TITLE
Ubuntu service doesn't notify systemd

### ISSUE THREAD

**Issue by brianegge** [codeproject/CodeProject.AI-Server#84]
**Area of Concern**
- [X] Server
- [ ] Behaviour of one or more Modules [provide name(s), e.g. ObjectDetectionYolo]
- [ ] Installer
- [ ] Runtime [e.g. Python3.7, .NET]
- [ ] Module packages [e.g. PyTorch)
- [ ] Something else

**Describe the bug**
deb installer doesn't enable service. 
Service is setup for notify, but service doesn't notify systemd

**Expected behavior**
For .deb install, docs should indicate the following are required
```
sudo systemctl enable codeproject.ai-server.service
sudo systemctl start codeproject.ai-server.service
```
The service description should not include the `notfiy` setting, as it seems this service doesn't do this. Type should be simple.
```
$ cat /usr/bin/codeproject.ai-server-2.3.3/codeproject.ai-server.service
[Unit]
Description=CodeProject.AI Server 2.3.3 application

[Service]
# Our app can notify systemd of status changes
Type=notify
```

**Screenshots**

**Your System (please complete the following information):**
 - CodeProject.AI Server version: 2.3.3
 - OS: 22.04.1
 - System RAM 32GB
 - GPU (if available) GeForce RTX 3080
 - GPU RAM (if available)  

**Additional context**
log:
Jan 22 08:19:14 gladiator-ubuntu systemd[1]: codeproject.ai-server.service: start operation timed out. Terminating.

**Comment by ahmetkipkip** [comment]
I have Same issue for 2.5.4 on ubuntu 20.04.  I made Type=simple to run continuously but lost handlers.

**Comment by ygelfand** [comment]
looked at the dotnet source, problem stems from how the systemd config is written, it can't fork off, as the parent pid is wrong then.  But couldn't find where/how the ubuntu package/systemd config is maintained to create a PR

**Comment by ChrisMaunder** [comment]
This has been fixed in 2.6.4 which will be published soon
